core_integrations/olarm

```diff
@@ -0,0 +1 @@
+../core_brands/olarm
```

Comment:
This seems incorrect. Is there a brand created @ the Core repository?

Which integrations are using this brand? As we only create brands if there are multiple integrations using the same branding.

../Frenck